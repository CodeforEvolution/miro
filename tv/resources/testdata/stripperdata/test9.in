In the first (of many) Jaqbacks, Stuart Langridge and Jono Bacon look at your emails and what&#8217;s going on in the Shot of Jaq community! The Jaqback shots appear outside the regular show schedule, so you won&#8217;t ever miss a show: email us your th