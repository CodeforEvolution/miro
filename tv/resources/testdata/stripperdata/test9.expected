(u'In the first (of many) Jaqbacks, Stuart Langridge and Jono Bacon look at your emails and what\u2019s going on in the Shot of Jaq community! The Jaqback shots appear outside the regular show schedule, so you won\u2019t ever miss a show: email us your th', [])